### DISCUSSION TITLE
Galera, o que estão sentindo do mercado mobile? RN x Flutter?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1648] [Perguntas e Respostas]
Fala galera, beleza?

Desculpem se estou postando algum fora do escopo, considero tanto o fronend web quanto o app mobile como o frontend da minha aplicação, na verdade sempre chamo ambos de clients.

Estou focado em web atualmente, porém assim que der um tempinho vou direcionar um pouco de tempo a aprender mais e praticar programação para mobile.

Pelo que estão sentindo do mercado e pelo que estão vendo, RN será que continua sendo bem forte no cross-plataform? Flutter será que ainda vai ser adotado como o principal neste quesito? Quais as opiniões de vocês sobre este assunto?

O que me aguçou bastante foram os dois vídeos abaixo:

-> Felipe Deschamps na previsão 12 citou que React Native manteria a frente segundo um artigo de um grande influencer e arquiteto de softwate influente estrangeiro.
vídeo: https://www.youtube.com/watch?v=5WwFXeHSZHE

-> Fluterando postou um vídeo citando quey Flutter está batendo React Native em popularidadae:
vídeo: https://www.youtube.com/watch?v=aC6krx-WZ3A&t=9s

O que vocês acham?

Abraços!

**Comment by kkkasio** [comment]
Fala @wilsonneto-dev  beleza cara?
O que faz sentido para você? React ou Flutter?
Tenho a certeza que, ambos são exelentes tecnolgoias. E na minha visão devemos olhar **nossos OBJETIVOS**. não seria interessante ser "**foda**" em ambos não acha? talvez a perda de tempo seria grande....

Creio que... você te realizar uma reflexão uma interna, e entender o que faz sentido para você!
Hoje eu estudo bastante react, porém fico antenado em oturas techs! quando o react parar de fazer sentido para o meu objetivo, eu irei mudar!

**Comment by wilsonneto-swe** [comment]
Show @kkkasio! Valeu pela opinião, vou estudar bastante react-native e usar ele em projetos pessoais, faz bem mais sentido para mim hoje. Mas to de olho no mercado geral, por isso queria opiniões.

Abraços!

**Comment by jpferreiradev** [comment]
As vezes eu acho complicado você ficar comparando um com o outro para ver qual que vai ficar na frente, eu particularmente estou estudando para trabalhar com Android nativo, e já me deu uma grande vontade de estudar um ai depois o outro, é bom ter um foco mesmo e um e vai, e é claro ficar antenado no que está acontecendo, são muitos "dependes" nessa área, o importante mesmo é focar, se não fica doido, kkkkk!

**Comment by caioincau** [comment]
Vejo Flutter ganhando muito mercado e rápida adoção de grandes empresas, enquanto vejo o RN diminuindo em grandes empresas, porém aumentando em pequenas que aproveitam o front-End developer para fazer ambos.

Meu feeling de mercado

**Comment by cwsouto** [comment]
Galera, vocês conhecem desenvolvedores Flutter no Espirito Santo?
